File: nestjs/notifications-service/src/common/messaging.constants.ts
export const MESSAGING = {
  exchange: 'pulse.events',
  queue: 'order.created',
  routingKey: 'order.created',
  eventTypeOrderCreated: 'OrderCreated',
  correlationHeader: 'x-correlation-id',
  retryCountHeader: 'x-retry-count',
  maxRetries: 5,
} as const;

File: dotnet/src/NotificationsService/NotificationsService.Domain/MessagingConstants.cs
namespace NotificationsService.Domain;

public static class MessagingConstants
{
    public const string ExchangeName = "pulse.events";
    public const string QueueName = "order.created";
    public const string RoutingKey = "order.created";
    public const string EventTypeOrderCreated = "OrderCreated";
    public const string CorrelationHeader = "x-correlation-id";
    public const string RetryCountHeader = "x-retry-count";
}
